tests/flows/steps/conftest.py

```
@@ -0,0 +1,47 @@
+@pytest.fixture
+def mock_teams_notification_manager(mocker):
+    return mocker.patch("swo_aws_extension.flows.steps.base.TeamsNotificationManager", spec=True)
+
+
+@pytest.fixture
+def mock_awsclient(mocker):
+    return mocker.Mock(spec=AWSClient)
+
+
+@pytest.fixture
+def context_with_agreement(agreement_factory, order_factory, config, mock_awsclient):
```

Ruff (0.14.10)

Unused function argument: `config`

(ARG001)